projects-2.13.md

```diff
@@ -67,15 +69,15 @@ Add in sbt using `libraryDependencies += ...`:
     "com.lihaoyi"                            %% "upack"                   % "0.7.5"
     "com.lihaoyi"                            %% "upickle"                 % "0.7.5"
     "io.argonaut"                            %% "argonaut"                % "6.2.3"
-    "org.foundweekends"                      %% "knockoff"                % "0.8.12"
+    "org.foundweekends"                      %% "knockoff"                 % "0.8.12"
```

Comment:
```suggestion
    "org.foundweekends"                      %% "knockoff"                % "0.8.12"
```

Reply:
I purposely aligned those again.

Reply:
It's looking aligned in master to me: https://github.com/scala/make-release-notes/blob/2.13.x/projects-2.13.md

and not in your branch: https://github.com/larsrh/make-release-notes/blob/f8f9050ed5e6c14029272e10c9d84a86b117fe12/projects-2.13.md

Reply:
Oh. I have ligatures enabled. I take everything back :joy: